setup.py

```diff
@@ -47,13 +47,13 @@
             'black<=19.10b0',
             'flake8<=3.9',
             'isort~=5.0',
-            'readme-renderer~=26.0',
+            'readme-renderer~=28.0',
```

Comment:
```suggestion
            'readme-renderer~=28.0.0',
```
`readme-renderer~=28.0.` translates to `readme-renderer==28.*`. This has created problems in openwisp-notifications earlier. Please check [compatible release in PEP 440](https://www.python.org/dev/peps/pep-0440/#compatible-release) for more info.